### ISSUE TITLE
Brew doctor gives a `No developer tools installed` on Catalina and they are installed

### ISSUE THREAD

**Issue by ericvn** [Homebrew/brew#6610]
**Please note that we will close your issue without comment if you delete, do not read or do not fill out the issue checklist below and provide ALL the requested information. If you repeatedly fail to use the issue template, we will block you from ever submitting issues to Homebrew again.**

- [ x ] are reporting a bug others will be able to reproduce and not asking a question. If you're not sure or want to ask a question do so on our Discourse: https://discourse.brew.sh
- [ ] ran a `brew` command and reproduced the problem with multiple formulae? If it's a problem with a single, official formula (not cask) please file this issue at Homebrew/homebrew-core: https://github.com/Homebrew/homebrew-core/issues/new/choose. If it's a `brew cask` problem please file this issue at https://github.com/Homebrew/homebrew-cask/issues/new/choose. If it's a tap (e.g. Homebrew/homebrew-php) problem please file this issue at the tap.
- [ x ] ran `brew update` and can still reproduce the problem?
- [ x ] ran `brew doctor`, fixed all issues and can still reproduce the problem?
- [ x ] ran `brew config` and `brew doctor` and included their output with your issue?

# What you were trying to do (and why)

Run `brew doctor` 

# What happened (include command output)

<details>
  <summary>Command output</summary>
  <pre>
  
 brew doctor                                                                                                                                                          
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: No developer tools installed.
Install the Command Line Tools:
  xcode-select --install
  
  </pre>
</details>

Running the suggested command:

```
xcode-select --install                                                                                                                                                 
xcode-select: error: command line tools are already installed, use "Software Update" to install updates
```

There are no updates available.

# What you expected to happen

No warnings from `brew doctor`

Brew config output:
```
brew config                                                                                                                                                          
HOMEBREW_VERSION: 2.1.14-39-g48f54d1
ORIGIN: https://github.com/Homebrew/brew
HEAD: 48f54d17f8e41fc9588a500b44027547c4058bab
Last commit: 34 minutes ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 6eb3028f2c4a37ac8bb89e9adb6631f9ff70125d
Core tap last commit: 45 minutes ago
HOMEBREW_PREFIX: /usr/local
HOMEBREW_DEV_CMD_RUN: 1
HOMEBREW_VISUAL: nano
CPU: octa-core 64-bit kabylake
Homebrew Ruby: 2.6.3 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
Clang: N/A
Git: 2.23.0 => /usr/local/bin/git
Curl: 7.64.1 => /usr/bin/curl
Java: 1.8.0_221, 1.8.0_211, 1.8.0_201, 1.8.0_191
macOS: 10.15-x86_64
CLT: N/A
Xcode: N/A
```

**Comment by MikeMcQuaid** [comment]
> * [ ]  ran `brew config` and `brew doctor` and included their output with your issue?

**Comment by ericvn** [comment]
`brew config` output added. `brew doctor` was already included.

**Comment by MikeMcQuaid** [comment]
Please always read and fill out the issue template for future issues.

What happens if you open Software Update?

**Comment by ericvn** [comment]
Sorry about that. Was reading the issue and you closed it before I was able to get things updated. I know for my next issue (hoping there isn't one) in this repo what is needed.

The AppStore shows no updates, and the System Preferences->Software Updates shows no entries.

I typically run `brew doctor` daily and don't recall this warning yesterday, but I rebooted when the Mac 10.15 Supplemental update was applied so can't go back in the terminal to verify.

It could be related to applying yesterday's supplemental update.
